BGL/include/CGAL/boost/graph/properties.h

```diff
@@ -86,13 +91,37 @@ using boost::face_external_index;
 namespace CGAL{
 namespace helpers {
 
+//check that an existing map is initialized/valid
+template <class SimplexRange, class IndexMap>
+bool is_index_map_valid( std::size_t num_simplices,
+                        const SimplexRange& range,
+                        const IndexMap& idmap)
+{
+  
+  typedef typename boost::property_traits<IndexMap>::value_type Id_type;
+  Id_type max_id = static_cast<Id_type>(num_simplices);
+  std::vector<bool> indices(max_id + 1);
```

Comment:
Why max_id +1, the max_id that an element can have is size-1, here you seem to be at size+1?